.github/workflows/comment-command.yml

```diff
@@ -102,3 +102,87 @@ jobs:
           # Commit
           git commit -am "Regenerate pregenerated files"
           git push
+
+  update:
```

Comment:
This duplicates `.github/workflows/upstream-utils.yml`. Maybe you could dispatch to that workflow from here?